solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -151,27 +152,42 @@ contract RandomBeacon is IRandomBeacon, Ownable {
     ///         operator affected.
     uint256 public dkgMaliciousResultNotificationRewardMultiplier;
 
+    /// @notice Calculated gas cost for submitting a DKG result. This will
+    ///         be refunded as part of the DKG approval process. It is in the
+    ///         submitter's interest to not skip his priority turn on the approval,
+    ///         otherwise the refund of the DKG submission will be refunded to
+    ///         other member that will call the DKG approve function.
+    uint256 public dkgResultSubmissionGas = 235000;
+
+    // @notice Gas is meant to balance the DKG approval's overall cost. Can be
```

Comment:
```suggestion
    // @notice Gas meant to balance the DKG approval's overall cost. Can be
```

Same for other offsets below.

Reply:
You sure? I think the [correct form](https://www.collinsdictionary.com/dictionary/english/be-meant-to#:~:text=phrase,to%20announce%20it%20to%20everyone.) is "be meant to"

Reply:
Right but what I am trying to say is that we are describing this particular field. So maybe just: *Gas that is meant to...*"?